src/Stackctl/StackSpecYaml.hs

```diff
@@ -64,8 +66,19 @@ instance ToJSON StackSpecYaml where
 newtype ParametersYaml = ParametersYaml
   { unParametersYaml :: [ParameterYaml]
   }
+  deriving stock (Eq, Show)
   deriving newtype ToJSON
 
+instance Semigroup ParametersYaml where
+  ParametersYaml as <> ParametersYaml bs =
+    ParametersYaml
+      $ map (uncurry ParameterYaml)
+      $ KeyMap.toList
+      $ KeyMap.fromListWith (<>)
+      $ map (pyKey &&& pyValue)
```

Comment:
could you do `map (pyKey &&& (Last . pyValue))` here to achieve the same behaviro w/o putting Last in parametervalue type and/or explicitly telling it what to do?

Eh, then again, you'd have to `unLast` the values just after the fromListWith, so probably not worth it...